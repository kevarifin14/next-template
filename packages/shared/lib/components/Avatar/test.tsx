import { render } from '@testing-library/react';
import React from 'react';

import { Avatar } from '.';

describe('Avatar', () => {
  it('matches snapshot', () => {
    const { baseElement } = render(<Avatar src="" />);
    expect(baseElement).toMatchSnapshot();
  });
});
