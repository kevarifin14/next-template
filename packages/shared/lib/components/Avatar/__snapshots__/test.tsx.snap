// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar matches snapshot 1`] = `
<body>
  <div>
    <div
      class="relative rounded-full overflow-hidden flex items-center flex-shrink-0 h-10 w-10"
    >
      <img
        alt=""
        src=""
      />
    </div>
  </div>
</body>
`;
